src/utils/index.js

```diff
@@ -59,8 +59,8 @@ function isDefined(value) {
   return value !== undefined && value !== null
 }
 
-export function allFilesAccepted({ files, accept, minSize, maxSize, multiple }) {
-  if (!multiple && files.length > 1) {
+export function allFilesAccepted({ files, accept, minSize, maxSize, multiple,maxFiles }) {
+  if ((!multiple && files.length > 1) ||(multiple && maxFiles >= 1 &&  files.length > maxFiles) ) {
```

Comment:
```suggestion
export function allFilesAccepted({ files, accept, minSize, maxSize, multiple, maxFiles }) {
  if ((!multiple && files.length > 1) || (multiple && maxFiles >= 1 &&  files.length > maxFiles) ) {
```

Reply:
I couldn't find a way to run linter to format staged files (husky should handle this, but it didn't), and there is no prettier config file
please give me a hint about it.